accounting_reporting_core/src/main/java/org/cardanofoundation/lob/app/accounting_reporting_core/domain/entity/report/ReportEntity.java

```diff
@@ -118,6 +119,15 @@ public String getId() {
     @Setter
     private LocalDate date;
 
+    @Embedded
+    @AttributeOverrides({
+            @AttributeOverride(name = "primaryBlockchainType", column = @Column(name = "primary_blockchain_type")),
+            @AttributeOverride(name = "primaryBlockchainHash", column = @Column(name = "primary_blockchain_hash"))
+    })
+    @Getter
+    @Setter
+    private LedgerDispatchReceipt ledgerDispatchReceipt;
```

Comment:
The whole object here is https://github.com/nullable actually, not individual fields. In DB individual fields but in Java, the whole object.